### DISCUSSION TITLE
Custom Button Sizing

### DISCUSSION THREAD

**Discussion by rozza591** [Clooos/Bubble-Card#329] [Questions about config, custom styles and templates]
I may just be trying to impliment this incorrectly but when I try to change the height of the button slider with css styling it only changes the height of the card it sits in, not the slider itself.

**Comment by dts-1990** [comment]
I'd love to see this implemented too. Currently, all buttons have the same size; changing the Card layout in Styling option doesn't make the button much bigger than I want.

**Comment by ahume** [comment]
👍🏻 For this. I wondered about re-implementing by mushroom based wall mounted tablet dashboard, but I can't re-create the large hit targets I have with mushroom.
![Screenshot 2024-12-02 at 11 44 50](https://github.com/user-attachments/assets/cf45ae92-979b-4ffd-922d-274bde1e161f)

**Reply by daproject85** [reply]
how do you make a button with just an icon ?

**Comment by MrBearPresident** [comment]
I'm working on that: 

#965 

Meanwhile use this code in the style element: 
```
  .bubble-button-card-container{
    height: calc( var(--row-height) * var(--row-size) + var(--row-gap) * ( var(--row-size) - 1 )) !important;
  }
```


---

<sub>If this solves your issue, please mark this as solved.</sub>

**Comment by john-ezra** [comment]
Did this ever get any traction or a solution?  I'm in the same situation.
